### PR TITLE
Add Elastic APM SourceMap Plugin for Webpack RUM agent list

### PR DESCRIPTION
**Pull request by jahtalab**

### CODE REVIEW

File: apm-agent-rum-js/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Elastic APM JavaScript Real User Monitoring (RUM) agent
+
+This is a collection of code and recipes to be used in conjunction with [Elastic APM JavaScript RUM agent](https://github.com/elastic/apm-agent-rum-js). The code here is not necessarily maintained by the core Elastic APM Team. 
+
+
+## Elastic APM SourceMap Plugin for Webpack
+[Elastic APM SourceMap Plugin](https://github.com/wuct/elastic-apm-sourcemap-webpack-plugin/) is a Webpack plugin that uploads sourcemaps generated by Webpack to a Elastic APM server.
```
Comment (watson) [inline apm-agent-rum-js/README.md]:
```suggestion

- [elastic-apm-sourcemap-webpack-plugin](https://github.com/wuct/elastic-apm-sourcemap-webpack-plugin/) is a Webpack plugin that uploads sourcemaps generated by Webpack to an Elastic APM Server.
```

File: apm-agent-rum-js/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
```
Comment (watson) [inline apm-agent-rum-js/README.md]:
```suggestion
```

File: apm-agent-rum-js/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Elastic APM JavaScript Real User Monitoring (RUM) agent
+
+This is a collection of code and recipes to be used in conjunction with [Elastic APM JavaScript RUM agent](https://github.com/elastic/apm-agent-rum-js). The code here is not necessarily maintained by the core Elastic APM Team. 
+
+
```
Comment (watson) [inline apm-agent-rum-js/README.md]:
```suggestion
```

File: apm-agent-rum-js/README.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+## Elastic APM SourceMap Plugin for Webpack
```
Comment (watson) [inline apm-agent-rum-js/README.md]:
```suggestion
## Tools & Libraries
```

Comment (watson) [reply apm-agent-rum-js/README.md]: @jahtalab you can probably come up with a better headline 😅

File: apm-agent-rum-js/README.md (hunk 1 above)
Comment (watson) [reply apm-agent-rum-js/README.md]: I'm not sure what exactly the linked text should if not the name of the module. Else we just end up repeating our selfs with what's written after the link 🤷‍♂️

**Comment by bmorelli25** [comment]
The renaming of the `js-base` folder will break links from the `apm` and `apm-dev` repos. Let me open PRs there to change the links first, then you can merge everything at the same time tomorrow.

**Pull request merged**